agrirouter-api-java-impl/src/main/java/com/dke/data/agrirouter/impl/onboard/cloud/OnboardingServiceImpl.java

```diff
@@ -31,198 +31,198 @@
-
-        List<OnboardingResponse> responses = new ArrayList<>();
-        if (fetchMessageResponses.isPresent()) {
-            DecodeMessageResponse decodedMessageQueryResponse =
-                    this.decodeMessageService.decode(
-                            fetchMessageResponses.get().get(0).getCommand().getMessage());
-            if (decodedMessageQueryResponse.getResponseEnvelope().getResponseCode()
-                    == HttpStatus.SC_BAD_REQUEST) {
-                MessageOuterClass.Message message =
-                        this.decodeMessageService.decode(
-                                decodedMessageQueryResponse.getResponsePayloadWrapper().getDetails().getValue());
-                throw new CouldNotOnboardVirtualCommunicationUnitException(message.getMessage());
-            } else {
-                if (decodedMessageQueryResponse.getResponseEnvelope().getType()
-                        == Response.ResponseEnvelope.ResponseBodyType.CLOUD_REGISTRATIONS
-                        && decodedMessageQueryResponse.getResponseEnvelope().getResponseCode()
-                        == HttpStatus.SC_CREATED) {
-                    CloudVirtualizedAppRegistration.OnboardingResponse onboardingResponse =
-                            this.decode(
-                                    decodedMessageQueryResponse.getResponsePayloadWrapper().getDetails().getValue());
-                    onboardingResponse
-                            .getOnboardedEndpointsList()
-                            .forEach(
-                                    endpointRegistrationDetails -> {
-                                        OnboardingResponse internalOnboardingResponse = new OnboardingResponse();
-                                        internalOnboardingResponse.setSensorAlternateId(
-                                                endpointRegistrationDetails.getSensorAlternateId());
-                                        internalOnboardingResponse.setCapabilityAlternateId(
-                                                endpointRegistrationDetails.getCapabilityAlternateId());
-                                        internalOnboardingResponse.setDeviceAlternateId(
-                                                endpointRegistrationDetails.getDeviceAlternateId());
-                                        internalOnboardingResponse.setAuthentication(
-                                                parameters.getOnboardingResponse().getAuthentication());
-                                        internalOnboardingResponse.setConnectionCriteria(
-                                                parameters.getOnboardingResponse().getConnectionCriteria());
-                                        responses.add(internalOnboardingResponse);
-                                    });
-                }
-            }
+  private final EncodeMessageService encodeMessageService;
```

Comment:
Are those changes format changes? If they are it would be nice to create an additional PR with format changes only. Then we are able to see what has change on a functional level more easily.